sohumrao/JIC-4103-Beauty-Consulting-App
# Stylist Photo Upload and Generic Photo Display Component

- Changed ProfileImage component to ImageUploadButton that is solely for uploading, not displaying
- Added dynamic ProfilePhotoDisplay component that takes in photo data and styling as props
- Added ProfilePhotoDisplay and ImageUploadButton to BusinessInfoPage
- Added ProfilePhotoDisplay to all pages that display photos. Style props are designed to dynamically allocate photo size based on screen size to avoid issues with small screens
- Deleted profile picture from client profile view while that screen is getting fixed

To test:

- Log in as a stylist or create a stylist account
- If account has no profile photo, there should be a placeholder. If a photo exists it should be displayed
- Test edit button and upload a new profile photo
- Make sure this photo persists in the DB by logging out / logging back in
- Make sure photos display correctly on all pages
